src/main/java/jenkins/plugins/git/GitRemoteHeadRefAction.java

```diff
@@ -62,6 +59,4 @@ public String toString() {
                 ", name='" + name + '\'' +
                 '}';
     }
-
```

Comment:
Please don't make these types of whitespace changes in the pull request.  They risk merge conflicts with other pull requests and complicate code review

Reply:
done